common/djangoapps/django_sudo_helpers/tests/test_third_party_auth_links.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+"""
+Tests django_sudo_helpers templatetags to create third party auth links.
+"""
+import unittest
+from mock import patch
+from django.conf import settings
+from social.apps.django_app.default import models as social_models
+from django.test import TestCase
+from django.test.client import RequestFactory
+
+from third_party_auth.tests import testutil
+from django_sudo_helpers.templatetags import third_party_auth_links
+
+
+@unittest.skipUnless(testutil.AUTH_FEATURE_ENABLED, 'third_party_auth not enabled')
+class ThirdPartyAuthLinkTests(testutil.TestCase, TestCase):
```

Comment:
No need to inherit from `django.test.TestCase`, as `testutil.TestCase` already does this.